### PR TITLE
fitdla and fitlls guis - adding options (see below)

### PR DESCRIPTION
**Pull request by marijana777**
1) making arguments masking and norm optional
2) adding option conti_file='from_spectrum' to use continuum data from the original spectrum

### CODE REVIEW

File: docs/fitdla.rst [hunk 1]
```diff
@@ -40,6 +40,7 @@ script `pyigm_fitdla`.  Here is the usage::
                             Input DLA Fit file
       --conti_file CONTI_FILE
                             Input continuum spectrum
+                            Option 'from_spectrum': continuum data will be used from the original (in_file) spectrum.
```
Comment (profxj) [inline docs/fitdla.rst:43]: Add that 100 spline points will be added uniformly 
along the input continuum.

**Review by profxj — APPROVED** [review]
One minor request.

File: docs/fitdla.rst (hunk 1 above)
Comment (marijana777) [reply docs/fitdla.rst:43]: Added.

**Comment by profxj** [comment]
merging

**Pull request merged**